### PR TITLE
Feature/conda env bcftools

### PR DESCRIPTION
**Pull request by henrikstranneheim**
…ftools when checking references

### This PR adds | fixes:

- See above

### How to test:

- Automatic and continuous test suite

### Expected outcome:
- [ ] Installation, unit and integration test suite pass

### Review:
- [ ] Code review
- [ ] Tests pass

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes. If applicable record manual test results in PR header
- [ ] **MINOR** - when you add functionality in a backwards compatible manner. If applicable record manual test results in PR header
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: lib/MIP/Check/Reference.pm [hunk 1]
```diff
@@ -102,7 +110,8 @@ sub check_if_processed_by_vt {
         my $unset_cmd = join $SPACE, gnu_unset( { bash_variable => q{MIP_BIND}, } );
 
         ## Detect if vt program has processed reference
-        my $ret = `$export_cmd; bcftools view $reference_file_path | $regexp; $unset_cmd`;
+        my $ret =
+`$export_cmd; $bcftools_binary_path view $reference_file_path | $regexp; $unset_cmd`;
```
Comment (jemten) [inline lib/MIP/Check/Reference.pm:114]: let's use the sub instead of backticks.

**Comment by henrikstranneheim** [comment]
Good point! I think it is being handled here
```
## Export MIP_BIND to bind reference path to htslib sif in proxy bin
        my $export_cmd = join $SPACE,
          gnu_export( { bash_variable => q{MIP_BIND} . $EQUALS . $reference_file_path } );

        ## Unset MIP_BIND after system parsing
        my $unset_cmd = join $SPACE, gnu_unset( { bash_variable => q{MIP_BIND}, } );

        ## Detect if vt program has processed reference
        my $ret =
`$export_cmd; $bcftools_binary_path view $reference_file_path | $regexp; $unset_cmd`;
```
Remind me when I make the PR to master that this is included

File: lib/MIP/Check/Reference.pm (hunk 1 above)
Comment (henrikstranneheim) [reply lib/MIP/Check/Reference.pm]: Done

**Comment by jemten** [comment]
> Good point! I think it is being handled here
> 
> ```
> ## Export MIP_BIND to bind reference path to htslib sif in proxy bin
>         my $export_cmd = join $SPACE,
>           gnu_export( { bash_variable => q{MIP_BIND} . $EQUALS . $reference_file_path } );
> 
>         ## Unset MIP_BIND after system parsing
>         my $unset_cmd = join $SPACE, gnu_unset( { bash_variable => q{MIP_BIND}, } );
> 
>         ## Detect if vt program has processed reference
>         my $ret =
> `$export_cmd; $bcftools_binary_path view $reference_file_path | $regexp; $unset_cmd`;
> ```
> 
You're right! Then there should be no issues :)

**Pull request merged**